app/resources/code.yamil

```diff
@@ -72,3 +72,6 @@ def match_letters(x: string) -> bool = {
 match_letters("Hey")
 match_letters("C")
 match_letters("Haskell")
+
+def lambda(x: string) -> bool = {
```

Comment:
It is good to have seen that this line makes the correct validation for reserved names, but it would be better to not have it in the main file to keep advancing.